packages/atoms/src/InputWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import React, { ReactNode } from 'react';
+import { View } from 'react-native';
+import { Box } from './Box';
+import { Icon } from './Icon/Icon';
+import { IconType } from './Icon/iconTypes';
+
+type InputWrapperProps = {
+    children: ReactNode;
+    leftIconType: IconType;
+    isClearable?: boolean;
+    clearIcon?: ReactNode;
+} & InputStyleProps;
+
+type InputStyleProps = {
+    isFocused: boolean;
+};
+const inputWrapperStyle = prepareNativeStyle<InputStyleProps>((utils, { isFocused }) => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: 48,
+    borderWidth: utils.borders.widths.sm,
+    borderColor: utils.colors.gray200,
+    backgroundColor: utils.colors.gray200,
+    borderRadius: utils.borders.radii.basic,
+    paddingLeft: 14,
+    paddingRight: 14.25,
+    extend: [
+        {
+            condition: isFocused,
+            style: {
+                borderColor: utils.colors.gray500,
+            },
+        },
+    ],
+}));
+
+export const InputWrapper = ({
```

Comment:
While it's true that the search input and others share a lot of similarities, a "wrapper" is not very flexible. I have implemented `InputStyles` for the web and that approach is more suitable for sharing between `Select` and `Input` for example, since on the inside the structure is different but the styles are the same. Search input has a different height, always has the magnifying glass, has no label, can be cleared (which makes little sense in the wrapper) and the component should reflect those differences, not a wrapper. So I would not have a separate wrapper component, since this exact one is unlikely to fit any other inputs from figma and trying to do so is too extra.

Reply:
Yea, this wrapper is not what I wanted, please remove both of icons and leave only `children` here, that will make component much more flexible.

Reply:
Actually I spend little bit more time testing it and we will have try different approach. As you can see real TextInput is very small part of this SearchInput component:
<img width="365" alt="Screenshot 2022-05-19 at 21 07 09" src="https://user-images.githubusercontent.com/5837757/169384159-dbb07bb3-3d13-4cc9-86ed-9cdfb8679c26.png">

That is making UX significantly worse because you have very limited space where you can click and it will focus input. I think correct behaviour is when you click anywhere into input including search icon, it will focus input. To achieve that you will probably need to kill this component and move most of it's styled directly to TextInput component. Also you will probably need to add Pressable to search icon, and focus TextInput when icon is pressed using TextInput ref.

Reply:
Yeah, good catch! It's fixed now. I moved the wrapper back to the input component and when I do the other input / dropdown component, I will have a look if there are similarities that could be extracted into wrapper that will be more flexible. 

Added TouchableWithoutFeedback to wrap the whole input component which should take care of focusing the input. 
[74ed1c1](https://github.com/trezor/trezor-suite/pull/5433/commits/74ed1c1dfb41f1b97d929e2fbfc12f74b6f8c8f6)

Reply:
Can you just add paddings to the input? So that it's all touchable (except for the close button) and the icon is below? Idk if that absolute positioning is possible in RN. But `TouchableWithoutFeedback` makes sense to me :)

Reply:
And also what about having input styles separately like on desktop?

Reply:
Separating some core styles or maybe creating some BaseInput component makes sense, but I think it will be better to do that when we will going to actually create Input component, let's keep this PR simple.